airflow/jobs.py

```diff
@@ -708,22 +708,28 @@ def manage_slas(self, dag, session=None):
             Blocking tasks:
             <pre><code>{blocking_task_list}\n{bug}<code></pre>
             """.format(bug=asciiart.bug, **locals())
-            emails = []
+            emails = {'email': [], 'email_cc': [], 'email_bcc': []}
             for t in dag.tasks:
```

Comment:
Can we refactor this a bit? Separating the logic that you've added to maintain backward compatible to a new function and add a test for this? This will also make it more transparant. What do you think.